modules/installing-ocp-agent-iscsi-files.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_with_agent_based_installer/installing-using-iscsi.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="installing-ocp-agent-iscsi-files_{context}"]
+= Creating the installation files
+
+Use the following procedure to generate the ISO image and create an iPXE script to upload to your iSCSI target.
+
+.Procedure
+
+. Create the agent image by running the following command:
++
+[source,terminal]
+----
+$ openshift-install --dir <install_directory> agent create image
+----
+
+. Create an iPXE script by running the following command:
++
+[source,terminal]
+----
+$ cat << EOF > agent.ipxe
+!ipxe
+set initiator-iqn <iscsi_initiator_base>:\${hostname}
+sanboot --keep iscsi:<iscsi_network_subnet>.1::::$<iscsi_initiator_base>:\${hostname}
```

Comment:
The scsi_initiator_base do not have to be the same (but can be). We should make them different to make it clearer.
The one at line 26 is the name of the iSCSI initiator (resides on the host). The one at line 27 is the iSCSI target name.